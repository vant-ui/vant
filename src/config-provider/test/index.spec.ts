import { ConfigProvider } from '..';
import { mount } from '../../../test';

test('should render tag prop correctly', () => {
  const wrapper = mount(ConfigProvider, {
    props: {
      tag: 'section',
    },
  });
  expect(wrapper.html()).toMatchSnapshot();
});
