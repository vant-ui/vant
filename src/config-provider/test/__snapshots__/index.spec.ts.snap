// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render tag prop correctly 1`] = `
<section class="van-config-provider">
</section>
`;
